scripts/install/opt/seagate/cortx/motr/config/motr_cfg.sh

```diff
@@ -164,14 +164,24 @@ do_m0provision_action()
 
     msg "Configuring host [`hostname -f`]"
 
-    MD_DEVICE=$(lvs -o lv_path | grep "lv_raw_metadata" | head -1)
+    MD_DEVICE_NODE1=$(lvs -o lv_path | grep "lv_raw_metadata" | grep srvnode-1 | head -1)
+    MD_DEVICE_NODE2=$(lvs -o lv_path | grep "lv_raw_metadata" | grep srvnode-2 | head -1)
     if [[ $? -eq 0 ]];then
-        LVM_SIZE=$(lvs $MD_DEVICE  -o LV_SIZE \
+        LVM_SIZE1=$(lvs $MD_DEVICE_NODE1  -o LV_SIZE \
               --noheadings --units b --nosuffix | xargs)
 
+        LVM_SIZE2=$(lvs $MD_DEVICE_NODE2  -o LV_SIZE \
+              --noheadings --units b --nosuffix | xargs)
+
+	if [ $LVM_SIZE1 -le $LVM_SIZE2 ] ; then
```

Comment:
if [ "$LVM_SIZE1" -le "$LVM_SIZE2" ] ; then

double quote the shell variables.

Reply:
This are interger values, do we still have to put them inside quotes